File: ServerSpace/src/main/java/Models/ServLista/ServListaModel.java
package Models.ServLista;


import com.fasterxml.jackson.annotation.JsonProperty;
import EntityLayer.ServListaEntity;
public class ServListaModel {
    

    public ServListaModel() {
        this.listaId = 0;
        this.nombre = "";
    }

    public ServListaModel(ServListaEntity Ent) {
        this.listaId = Ent.getListaId();
        this.nombre = Ent.getNombre();
    }
    @JsonProperty("ListaId")
    private int listaId = 0;

    public int getListaId() {
        return listaId;
    }


    @JsonProperty("Nombre")
    private String nombre = "";

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

}

File: Space/Space.Server/Model/ServLista/ServListaModel.cs
using Space.EntityLayer;
using System.Text.Json.Serialization;

namespace Space.Server
{
    public class ServListaModel
    {
        public ServListaModel()
        {
            this.ListaId = 0;
            this.Nombre = String.Empty;
        }
        public ServListaModel(ServListaEntity Ent)
        {
            this.ListaId = Ent.ListaId;
            this.Nombre = Ent.Nombre;
        }
        [JsonPropertyName("ListaId")] public Int32 ListaId { get; set; }
        [JsonPropertyName("Nombre")] public String Nombre { get; set; }
    }
}
